ios/RCTWeChat.m

```diff
@@ -38,13 +37,17 @@ - (void)dealloc
     [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
-- (BOOL)handleOpenUrl:(NSURL *)aUrl
+- (BOOL)handleOpenURL:(NSNotification *)aNotification
 {
-    if ([WXApi handleOpenURL:aUrl delegate:self])
+    NSString * aURLString =  [aNotification userInfo][@"url"];
+    NSURL * aURL = [NSURL URLWithString:aURLString];
+
+    if ([WXApi handleOpenURL:aURL delegate:self])
     {
```

Comment:
If we use NSNotification as the first argument, There is no aURL object at the top of function block.

Please refer the line 45 of "/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.m":
```
+ (BOOL)application:(UIApplication *)application
            openURL:(NSURL *)URL
  sourceApplication:(NSString *)sourceApplication
         annotation:(id)annotation
{
  NSDictionary<NSString *, id> *payload = @{@"url": URL.absoluteString};
  [[NSNotificationCenter defaultCenter] postNotificationName:RCTOpenURLNotification
                                                      object:self
                                                    userInfo:payload];
  return YES;
}
```